plugins/logs/plugin_test.go

```diff
@@ -211,3 +357,44 @@ func (t *testServer) start() {
 func (t *testServer) stop() {
 	t.server.Close()
 }
+
+// getRandomString generates a random string of A-Z chars
+func getRandomString() string {
+	bytes := make([]byte, 15)
+	for i := 0; i < 15; i++ {
+		randomInt := 65 + rand.Intn(90-65)
+		bytes[i] = byte(randomInt)
+	}
+	return string(bytes)
+}
+
+func getValue(key string) string {
```

Comment:
I don't see a lot of value in making one function handle method, user, and path. I would probably create three separate functions and then call them explicitly.

Reply:
Done